{"links": {"self": {"href": "data/repositories/ignitionrobotics/ign-transport/issues/71/comments/40190329.json"}, "html": {"href": "#!/ignitionrobotics/ign-transport/issues/71#comment-40190329"}}, "issue": {"links": {"self": {"href": "data/repositories/ignitionrobotics/ign-transport/issues/71.json"}}, "type": "issue", "id": 71, "repository": {"links": {"self": {"href": "data/repositories/ignitionrobotics/ign-transport.json"}, "html": {"href": "#!/ignitionrobotics/ign-transport"}, "avatar": {"href": "data/bytebucket.org/ravatar/{4249390a-7f55-404b-990d-817ba94cc7ac}ts=1533306"}}, "type": "repository", "name": "ign-transport", "full_name": "ignitionrobotics/ign-transport", "uuid": "{4249390a-7f55-404b-990d-817ba94cc7ac}"}, "title": "Exception sending message in Docker"}, "content": {"raw": null, "markup": "markdown", "html": "", "type": "rendered"}, "created_on": "2017-09-29T09:15:38.185346+00:00", "user": {"display_name": "Hamza Merzi\u0107", "uuid": "{2100f54d-6bc6-4db0-ada5-db7be9320606}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B2100f54d-6bc6-4db0-ada5-db7be9320606%7D"}, "html": {"href": "https://bitbucket.org/%7B2100f54d-6bc6-4db0-ada5-db7be9320606%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:172a2e1e-2bf2-4054-9b9b-9d541ddc8e4e/ab842f18-c8ca-4a0c-a82a-6d4d39ffdb37/128"}}, "nickname": "hamzaethz", "type": "user", "account_id": "557058:172a2e1e-2bf2-4054-9b9b-9d541ddc8e4e"}, "updated_on": null, "type": "issue_comment", "id": 40190329}